### DISCUSSION TITLE
_ImportError: cannot import name 'collections_abc' from 'six.moves' (unknown location)_

### DISCUSSION THREAD

**Discussion by lukmannhaqeem** [gee-community/qgis-earthengine-plugin#121] [General]
Hi there,

I'm new to Google Earth Engine as well as Python. When I ran the script below in QGIS (3.10), I got an error. I'm not sure exactly what the issue is, but perhaps the program cannot locate the file's library. I appreciate your help in fixing this issue. 

**import ee**
_ImportError: cannot import name 'collections_abc' from 'six.moves' (unknown location)_

**Comment by XavierCLL** [comment]
Hi @lukmannhaqeem 

Are you using the latest version of the plugin? That error was already fixed check the issue #67
